behave/features/steps/frontend_assignment.py

```diff
@@ -494,6 +513,24 @@ def wait_until_dashboard_page_is_loaded(*args):
     wait.until(wait_until_dashboard_page_is_loaded)
 
 
+@step("the case does not show up on the call centre dashboard ooh")
+def step_impl_case_removed_from_list_ooh(context):
+    dashboard_url = "http://clafrontendooh:8000/call_centre/?ordering=-modified&page=1"
```

Comment:
maybe declare CLA_FRONTEND_OOH_URL in constants or wherever this variable is declared?

Reply:
Amended